components/icons/ApplePodcasts.vue

```diff
@@ -8,5 +8,4 @@
NTYEmjXYAQCAEgEJqWOm96iMraAcA5azjjOD2t3fSDgAIhABQeKsvres6K0Q7AChPqi5f8YEeWWXCMwACIQAUx1V/vYKxWADK0zmva2tcTUlkAJVEpFIpWgFAZTl2/8xP33+EdgBQVjq3hl761XXUQwZQWeghBFB5Vl1cd+aftNAOAMpHfY/3+k+vIg0CqDjU6wNQkba/rWvwydjEoQRNAaAcbHtLB6PZAVQieggBVCTNJ7/wc6trOz00BYCS2/62znVXNtAOACoRcwgBVLDDd4Vv/dBR2gFACZ12XeO1n1hJOwCoUPQQAqhga6+oZ70vACXUuMp30Tu4CgEgEAJAiZz3+vazXtlKOwBYesEW/YYvrg026zQFAAIhAJTMBW/sWHF+De0AYCmpHvnaj6+saScNAiAQAkBJeYLKi76wpvucEE0BYGkIWVz7iZVcdgAQCAGgLKi67Nyc1XZQdBTAUrjiAz1rL6+nHQBUAaqMAqgesyOZH73n0NTxJE0BoHgueWf3Oa9pox0AVAd6CAFUj5p2/YWfW+0NqTQFgCLZ9qZO0iAAAiEAlKmGXt/Lb9rQ0OulKQAU3IVv7tz2pg7aAUA1YcgogCo0O5L5/lv2Oz9pCgCFcsm7u895NX2DAKoNPYQAqlBNu/7ymzZ0baUAIIBC3C0p4qqP9JIGAVQleggBVK1MwvzB2w6O7ovTFABypnrl53969epL62gKAARCAKgwybDx8DeGn/zeGE0BIAf13d4rP9LbfTbDDQAQCAGgYu381+GH/mWIdgCwKG2nB15849pgs0ZTACAQAkBlO3rfzH1fHpju44oHYEHOfmXrea9r9zeQBgEQCAGgKkSG0z/74NGx/UwpBDDvvZEsrv5I76YXNdEUAAiEAFBVsgnrwZsGd32bKYUAfr/OM4IXvqWz59wamgIAgRAAqtOxB2bu/vu+yHCapgDwm7a8pOV57+vRvCzKBYBACABVLTqW+dXfHj+xM0JTAHCEWvSL39l92rWNNAUAAiEALBdH7g3/6pPHkxGDpgCWszNf3nrhmzt9dSpNAYBACADLS7g/9eDXBw/dOW1bNAaw7DSs9J3z6rbNL26mKQAQCAFg+ep/dPb+rwyM7qMAKbBcqB757Fe1bXtjp6ILWgMAgRAAljsza+/8t6Envz+Wjpq0BlDdes6ped77elrW+WkKACAQAsD/io5lnvrB+BPfHc0mGUIKVKHus0Lb39rVeWZQyHQMAgCBEAB+n8ljyYe/MXz47mkza9MaQHVoWuU7+9VtG5/fJCtEQQAgEALAqUz3pZ752cST3x/PJBhEClSwlRfWbv2T1u6za1hgEAAIhACwODMDqd0/nNh9C7EQqDw959ZsfXnr2svqaQoAIBACQO4iQ+mh3dF9t031PRphgQqgzNW0eza/qKlza6jnnBpaAwAIhABQMGP740/9YPzo/TOJ6SytAZSbzjOCW/6oZc0l9Z6QQmsAAIEQAIrCSYNH7g0P74kN7Y6G+7mKAqXkCSodW0JtpwdWXVzfvjFAgwAAgRAAlkg2ZQ3vjh7eET6xMzIzyOUUWDqaX+k6M7Tm0vpVF9eFWnUaBAAIhABQMkbaGtsf73t0dmYoNXEoMd2Xcv6GZgEKK9iit50WqO30tG8Mdp4RrGn30CYAQCAEgPJiW1JkODX8dGxkb3z46ejMQDo1a9AsQA4UTdR0eFrW+ts2Bru2hhpX+jxBJgcCAIEQACpHfDo7dSw5cTjhPCJDbjh0H1EzE2cdC+C3sp83pHpqVG9ICTRqjat8Lev8zWv9tZ1e5/+ifQCAQAgA1cC2JCNjpWeN2dHM7Eh6ZjA1O+b8IROfyibCWSclEhRR3VRd1vyyt1YNNGihVr2mzVPb4anp8NS264EmXfPKskr8AwACIQAsu6QoZZJmOmomZ4xU1HB+JqazqVkjGTGcv3T+YGYsM2tbhu3+NG3bcn86/+kkTPfPlrMFh+Q+LPvX2wQKd8sw90MI5w/uD1kI2f1PoUiyLGTF+YPzU3LinPNQNDfXeYKKN6Q62c9bo/jqNH+96nP/7P6N7lfo9wMAAiEAAAAAoGRkmgAAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAACIQAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAAgRAAAAAAQCAEAAAAABAIAQAAAAAEQgAAAAAgEAIAAAAACIQAAAAAAAIhAAAAAIBACAAAAAAgEAIAAAAAqsX/F2AAyYjiSd5wRzEAAAAASUVORK5CYII="/>
```

Comment:
oops, looks like a base 64 bitmap in here. Don't worry about this. I'll be gathering all the svg icons for the dropdown and will be housing them in the public folder.